data.sql

```diff
@@ -1,7 +1,17 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES ('Agumon','2020-02-03',0,true,10.23);
-INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES ('Gabumon','2018-11-15',2,true,8);
-INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES ('Pikachu','2021-01-07',1,false,15.04);
-INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES ('Devimon','2017-05-12',5,true,11);
+INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES 
+('Agumon','2020-02-03',0,true,10.23),
+('Gabumon','2018-11-15',2,true,8), 
+('Pikachu','2021-01-07',1,false,15.04), 
+('Devimon','2017-05-12',5,true,11);
 
+
+INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES 
+('Charmander','2020-02-08',0,false,11),
+('Plantmon','2018-10-15',2,true,5.7), 
+('Squirtle','1993-04-02',3,false,12.13), 
+('Angemon','2005-06-12',1,true,45),
```

Comment:
## OPTIONAL 
* I strongly suggest that you write your queries correctly as the highlighted weights are negative numbers, I mark this as optional because the previous CR skipped this notwithstanding fix this and your project would be perfect 💯 
![image](https://user-images.githubusercontent.com/52497246/214511044-59bb3b01-a7ae-4ae2-844b-bc7594a233de.png)

Reply:
Thank you 🙏 Update accordingly.